src/main/java/uk/gov/hmcts/reform/sscs/docmosis/service/DocmosisPdfGenerationService.java

```diff
@@ -42,14 +42,17 @@ public DocmosisPdfGenerationService(String pdfServiceEndpoint,
         checkArgument(!isNullOrEmpty(templateName), templateEmptyMessage);
         checkNotNull(documentHolder.getPlaceholders(), placeholdersEmptyMessage);
 
-        log.info("Making request to pdf service to generate pdf document with template {} "
-            + "and placeholders of size [{}]", templateName, documentHolder.getPlaceholders().size());
+        log.info("Making request to Docmosis pdf service to generate pdf document with template {} "
+            + "and placeholders of size [{}] to endpoint {}", templateName, documentHolder.getPlaceholders().size(),
+                this.pdfServiceEndpoint);
 
         try {
             ResponseEntity<byte[]> response =
                 restTemplate.postForEntity(pdfServiceEndpoint, request(templateName, documentHolder.getPlaceholders(), documentHolder.isPdfArchiveMode()), byte[].class);
             return response.getBody();
         } catch (Exception e) {
+            log.error("Failed to request PDF from Docmosis REST endpoint " + this.pdfServiceEndpoint + " with error " +
```

Comment:
I guess PdfGenerationException will have all the information as we are printing e.getMessage() and e
Do we still require log.error message here?

Reply:
I think so as it's valuable info for debugging and helping verify issues such as current one I'm facing. Presently the exception is not being logged explicitly and is down to the calling service to decide what to do with the exception. In tribunals case it's simply being converted to a 500 back through the api call with no logging of the actual error. Hence wanting to have a point of audit as close to origin as possible so all our services get it consistently in logs